src/components/course/course-header/CourseRunCards.jsx

```diff
@@ -28,18 +29,20 @@ const CourseRunCards = () => {
       hasEqualColumnHeights={false}
     >
       {availableCourseRuns.map((courseRun) => {
-        const redeemabilityForContentKey = redeemabilityPerContentKey?.find(r => r.contentKey === courseRun.key);
-        const redeemableSubsidyAccessPolicy = redeemabilityForContentKey?.redeemableSubsidyAccessPolicy;
+        const hasRedeemablePolicy = userSubsidyApplicableToCourse?.subsidyType === LEARNER_CREDIT_SUBSIDY_TYPE;
 
-        if (redeemableSubsidyAccessPolicy || missingUserSubsidyReason?.userMessage) {
+        // Render the newer `CourseRunCard` component when the user's
+        // subsidy (if any) is a policy OR if no disabled enroll reason
```

Comment:
clarifying question: "if no disabled enroll reason" feels like the opposite of `missingUserSubsidyReason?.userMessage`, but am I just misremembering what the semantics of that missing reason from can-redeem are?

Reply:
[copied from slack] ah, yep that comment was misleading; just updated it to:

> OR if there is a known disabled enroll reason.

`missingUserSubsidyReason` reflects the already-parsed disabled enroll reasons across all subsidy types (can-redeem or otherwise).